wl-js-analytics-adapter.sublime-snippet

```diff
@@ -0,0 +1,113 @@
+<snippet>
+  <content><![CDATA[
+/**
+ * @namespace WEBLINC.${1:adapterName}
+ */
+WEBLINC.analytics.registerAdapter('${1:adapterName}', function () {
+    'use strict';
+
+    var pageViewSent = false,
+
+        send = function (event, payload) {
+            if (WEBLINC.analytics.debug) {
+                console.log(arguments);
+            }
+
+            ${2://add your send logic here}
+        },
+
+        isLoaded = function () {
+            var scripts = _.toArray(document.scripts),
+
+            loaded = _.any(scripts, function (script) {
+                return script.src.match(WEBLINC.config.${1:adapterName}.scriptRegex);
```

Comment:
If this is 1-to-1, meaning that this adapter is a client for only one third party, i think that adding config is an additional step that isn't needed. 

Thoughts?